products/spanner/helpers/instance_helpers.py.erb

```diff
@@ -0,0 +1,33 @@
+def resource_to_create(module):
```

Comment:
In Terraform we separate these out into different files. These are small, so putting them together feels totally reasonable to me, but I wonder how much consistency we want between providers for this sort of thing? Having them separate also potentially allows for the ability to override the create function completely instead of just having two in the same file and deciding which one to call, although it's a bit trickier here because create and update are by default the same.

Reply:
I agree that we should have consistency between these things.

Maybe we should chat about these things at a higher level?